falass/readwrite.py

```diff
@@ -89,32 +90,30 @@ def read_pdb(self):
         be the last text in the 'TITLE' line, and the cell dimensions are taken from the 'CRYST1' line, and assumed to
         be orthorhomic. Non-orthorhomic cells are not necessarily supported.
         """
-        lines = line_count(self.pdbfile)
         print("Reading PDB file")
-        file = open(self.pdbfile, 'r')
-        percentage = 0
-        print_update(percentage)
-        atoms_each_timestep = []
-        for i, line in enumerate(file):
-            percentage_new = np.floor(i / lines * 100)
-            percentage = check_update(percentage, percentage_new)
-            if line[0:6] == "ATOM  ":
-                atoms_each_timestep.append(get_atom_position(self.cell[self.number_of_timesteps - 1][2], line,
-                                                                self.flip))
-            if "TITLE  " in line:
-                if self.number_of_timesteps == 0:
-                    self.number_of_timesteps, new_time = iterate_time(self.number_of_timesteps, line)
-                    self.times.append(new_time)
-                else:
-                    self.atoms.append(atoms_each_timestep)
-                    atoms_each_timestep = []
+        self.u = u = mda.Universe(self.pdbfile)
```

Comment:
I think ultimately you'll just need this one line, the lines below are so other things that expect the old way don't break for now